src/inputs/input-group.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'underscore';
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import InputContainer from './components/input-container';
+
+class InputGroup extends Component {
+    _onKeyUp = (e) => {
+        if (e.keyCode === 13 && e.shiftKey) {
```

Comment:
`const ENTER_KEY = 13` and `if (e.keyCode === ENTER_KEY)` might aid comprehension here.